web/src/containers/ExecutiveSummary.js

```diff
@@ -8,85 +9,112 @@ import { expandActivitySection } from '../actions/activities';
+
+import { jumpTo } from '../actions/navigation';
 import { selectApdYears } from '../reducers/apd.selectors';
-import { selectBudgetExecutiveSummary } from '../reducers/budget.selectors';
+import {
+  selectBudgetExecutiveSummary,
+  selectBudgetGrandTotal
+} from '../reducers/budget.selectors';
+
+class ExecutiveSummary extends PureComponent {
+  jump = to => () => {
+    const { jumpTo: action } = this.props;
+    action(to);
+  };
 
-const ExecutiveSummary = ({ data, years, expandSection }) => (
-  <Waypoint id="executive-summary-overview">
-    <Section isNumbered id="executive-summary" resource="executiveSummary">
-      <Waypoint id="executive-summary-summary" />
-      <Subsection
-        id="executive-summary-summary"
-        resource="executiveSummary.summary"
-      >
-        {data.map((d, i) => (
-          <div
-            key={d.key}
-            className="mb2 md-flex items-center alert alert-success"
+  render() {
+    const { data, total, years } = this.props;
+    return (
+      <Waypoint id="executive-summary-overview">
+        <Section isNumbered id="executive-summary" resource="executiveSummary">
+          <Waypoint id="executive-summary-summary" />
+          <Subsection
+            id="executive-summary-summary"
+            resource="executiveSummary.summary"
           >
-            <div className="p2 sm-m0 flex-auto">
-              {d.key !== 'all' ? (
-                <Fragment>
-                  <div className="h5">
-                    {t('activities.namePrefixAndNum', { number: i + 1 })}
-                  </div>
-                  <a
-                    href={`#activity-${d.key}`}
-                    className="h3 bold black"
-                    onClick={() => expandSection(d.key)}
-                  >
-                    {d.name || t('activities.noNameYet')}
-                  </a>
-                </Fragment>
-              ) : (
-                <div className="h3 bold">{d.name}</div>
-              )}
-              {d.descShort && <div>{d.descShort}</div>}
-            </div>
-            <div className="md-flex md-col-7">
-              {years.map(year => (
-                <div key={year} className="px2 py3 flex-auto">
-                  <div className="h5">{t('ffy', { year })}</div>
-                  <div className="h3 mono bold">
-                    <Dollars>{d.totals[year]}</Dollars>
-                  </div>
-                </div>
-              ))}
-              <div className="px2 py3 flex-auto">
-                <div className="h5">{t('executiveSummary.total')}</div>
-                <div className="h3 mono bold">
-                  <Dollars>{d.combined}</Dollars>
-                </div>
-              </div>
-            </div>
-          </div>
-        ))}
-      </Subsection>
+            {data.map((activity, i) => (
+              <Review
+                heading={`Activity ${i + 1}: ${activity.name ||
+                  t('activities.noNameYet')}`}
```

Comment:
In line with Laura's suggestion, semantically this should be:
```suggestion
                  t('activities.noNameYet')}`}
                headingLevel={4}
```

Reply:
lol BEAT